go.mod

```diff
@@ -42,6 +42,7 @@ require (
 	github.com/diskfs/go-diskfs v1.4.0
 	github.com/form3tech-oss/jwt-go v3.2.3+incompatible
 	github.com/go-openapi/errors v0.22.0
+	github.com/go-openapi/runtime v0.28.0
```

Comment:
out of curiosity, what triggered this change in `go.mod`?

Reply:
not sure, but I will reset it to what we have in master.